pkg/scheduler/apis/config/types.go

```diff
@@ -55,6 +56,14 @@ type KubeSchedulerConfiguration struct {
 	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
 	HardPodAffinitySymmetricWeight int32
 
+	// TopologySpreadConstraints is the cluster-level configuration for
+	// topology-based spreading.
+	// Constraints are applied to pods that don't define any in their PodSpec.
+	// Label selectors must be empty, as they are deduced from the pods'
+	// membership in Services, Replication Controllers, Replica sets or Stateful
+	// sets.
```

Comment:
We should also mention the default values if it's not specified.

Reply:
Done